### PR TITLE
Add metrics page

### LINKED ISSUE #95: Calculate useful metrics
@andesol commented on [Mon Oct 18 2021](https://github.com/canonical-web-and-design/workplace-engineering-squad/issues/166)

See miro board: https://miro.com/app/board/o9J_lwlCrzA=/

### PR DESCRIPTION
**Pull request by andesol**
## Done
- Added a page with metrics
## QA
- `dotrun`
- Go to http://localhost:8501/metrics
- Check tables are rendered
 ## Issue
Resolves #95

### CODE REVIEW

File: scripts/metrics/hiring_leads.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    for person, avg in avg_time:
+        if person in results:
+            results[person]["avg_time_to_reject"] = round(avg,2)
+        else:
+            results[person] = {
+                "avg": avg
+            }
+
+    filename = 'hiring_leads'
+    with open(f'scripts/metrics/data/{filename}.json', 'w') as outfile:
```
Comment (tbille) [inline scripts/metrics/hiring_leads.py:104]:
```suggestion
    with open('scripts/metrics/data/hiring_leads.json', 'w') as outfile:
```

Comment (andesol) [reply scripts/metrics/hiring_leads.py]: :+1:

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "setup-database": "python3 -c 'from webapp.database import setup_database; setup_database()'",
     "reset-database": "python3 -c 'from webapp.database import reset_database; reset_database()'",
     "check-ts": "tsc --noEmit"
-  },
+},
```
Comment (tbille) [inline package.json:25]: This can be reverted

**Review by tbille — APPROVED** [review]
Let's :ship:  it!

**Pull request merged**